cluster_regional.tf

```
@@ -56,6 +56,11 @@ resource "google_container_cluster" "primary" {
     }
   }
 
+  network_policy {
+    provider = "CALICO"
+    enabled  = "${var.network_policy ? 1 : 0}"
```

This should also be changed to true/false (and remember to regen).

```suggestion
    enabled  = "${var.network_policy ? true : false}"
```